.github/workflows/code-quality.yml

```
@@ -37,10 +40,13 @@ jobs:
+
+      - name: Enable Corepack
+        run: corepack enable
 
       - name: Install dependencies
-        run: npm ci
+        run: pnpm install --frozen-lockfile
 
       - name: Run TypeScript check
-        run: npm run typecheck
\ No newline at end of file
+        run: pnpm run typecheck
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)